MetadataExtractor/Formats/Xmp/XmpReader.cs

```diff
@@ -26,9 +26,11 @@ namespace MetadataExtractor.Formats.Xmp
     public sealed class XmpReader : IJpegSegmentMetadataReader
     {
         public const string JpegSegmentPreamble = "http://ns.adobe.com/xap/1.0/\0";
+        public const string JpegSegmentPreamblAlt = "://ns.adobe.com/xap/1.0/\0";
```

Comment:
Why is the URL scheme (e.g. http or https) omitted here? Is this dealing with a specific type of corruption? If so, why only this particular value? I'm concerned that we are over-fitting to a specific type of corruption here.